baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/ObjectsHashCodeUnnecessaryVarargsTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.errorprone;
+
+import org.junit.jupiter.api.Test;
+
+class ObjectsHashCodeUnnecessaryVarargsTest {
+
+    @Test
+    public void test() {
+        fix().addInputLines(
+                        "Test.java",
+                        "import java.util.Objects;",
+                        "public class Test {",
+                        "  void f(Object a, Object[] b, String[] c) {",
+                        "    Objects.hash(a);",
+                        "    Objects.hash(a, a);",
+                        "    Objects.hash(b);",
+                        "    Objects.hash(c);",
```

Comment:
these are both a bit sketchy cases where compiler should have issued lint message `warning: non-varargs call of varargs method with inexact argument type for last parameter;`